src/index.js

```diff
@@ -35,25 +35,102 @@ const QUERY = 'query'
 // AuthenticationRequest sending methods
 const REDIRECT = 'redirect'
 
+// Local storage keys
+const CURRENT_PROVIDER = 'solid.current-provider'
+const CURRENT_CREDENTIALS = 'solid.current-user'
+
+// Local storage key prefixes
+const RP_BY_PROVIDER = 'oidc.rp.by-provider.'
+const PROVIDER_BY_STATE = 'oidc.provider.by-state.'
+
 class ClientAuthOIDC {
   /**
    * @constructor
    * @param [options={}]
    * @param [options.window=Window] Optionally inject global browser window
    * @param [options.store=localStorage] Optionally inject localStorage
+   * @param [options.providerUri] {string} Previously selected provider uri
+   *   (typically loaded from storage)
+   * @param [options.redirectUri] {string} This app's callback redirect uri,
+   *   defaults to the current window's uri.
+   * @param [options.debug] {Function}
    */
   constructor (options = {}) {
     this.window = options.window || global.window
```

Comment:
I'm confused because`global` isn't defined in the browser, and calling `global.window` in the console throws a `ReferencError`. Did this work when you tested it in the browser?